public/static/js/admin.js

```diff
@@ -28,7 +28,13 @@ mciModule.controller('AdminSettingsController', ['$scope','$window', 'mciAdminRe
     var errorHandler = function(resp) {
       notificationService.pushNotification("Error saving settings: " + resp.data.error, "errorHeader");
     }
-    mciAdminRestService.saveSettings($scope.Settings, { success: successHandler, error: errorHandler });
+    // temporary workaround until the admin page has components for every part of the config
```

Comment:
can you throw a TODO and ticket or epic number here please.

Reply:
done